Cargo.toml

```diff
@@ -2,14 +2,14 @@
 name = "xmas-elf"
 version = "0.2.0"
 authors = ["Nick Cameron <ncameron@mozilla.com>"]
-description = "Library for parsing and navigatin ELF data; zero-allocation, type-safe."
-license = "Apache-2.0/MIT"
+description = "Library for parsing and navigating ELF data; zero-allocation, type-safe."
+license = "Apache-2.0 OR MIT"
 repository = "https://github.com/nrc/xmas-elf"
+keywords = ["elf", "parser", "binary"]
 
 [dependencies]
 zero = "0.1"
 
 [[bin]]
 name = "xmas_elf"
-publish = "false"
```

Comment:
I think this is required otherwise we publish the bin (which is only meant for testing) to crates.io instead of the lib